tests/ui/suggestions/incorrect-variant-in-arg.rs

```diff
@@ -0,0 +1,57 @@
+// Regression test for #146586
+
+//@ only-linux
+//@ compile-flags: --error-format=human --color=always
```

Comment:
```suggestion
```

I know this is a WIP PR but please don't keep this test Linux exclusive (that's just worse DX for Windows rustc devs) and don't keep it as an SVG test, there's no reason to add a 1100 line SVG for this diagnostic.